cobbler/api.py

```diff
@@ -1288,18 +1288,46 @@ def validate_autoinstall_files(self, logger=None):
 
     # ==========================================================================
 
-    def sync(self, verbose: bool = False, logger=None):
+    def sync(self, verbose: bool = False, what: bool = [], logger=None):
```

Comment:
I was brain afk apparently when doing that line. please change to `list` obviously.